scripts/rabbitmq-integration-test-2015.sh

```diff
@@ -0,0 +1,283 @@
+if [ $? != 0 ]
+then
+  fatal 1 "could activate virtualenv"
+fi
+export PYTHONPATH=.
+echo " Done."
+
+echo -n "INFO: setting up database..."
+socorro setupdb --dropdb --force --createdb > setupdb.log 2>&1
+if [ $? != 0 ]
+then
+  fatal 1 "setupdb_app.py failed, check setupdb.log"
+  cat setupdb.log
+fi
+echo " Done."
+popd >> setupdb.log 2>&1
+
+# ensure rabbitmq is really empty and no previous failure left garbage
+cleanup_rabbitmq
+
+echo -n "INFO: setting up 'weekly-reports-partitions' via crontabber..."
+python socorro/cron/crontabber_app.py \
+    --job=weekly-reports-partitions \
+    --force \
+    >> setupdb.log 2>&1
+if [ $? != 0 ]
+then
+  fatal 1 "crontabber weekly-reports-partitions failed, check setupdb.log"
+fi
+echo " Done."
+
+echo -n "INFO: configuring backend jobs..."
+for p in collector processor middleware
+do
+  cp config/${p}.ini-dist config/${p}.ini
+  if [ $? != 0 ]
+  then
+    fatal 1 "copying default config for $p failed"
+  fi
+  # ensure no running processes
+  fuser -k ${p}.log > /dev/null 2>&1
+done
+echo " Done."
+
+echo -n "INFO: starting up collector, processor and middleware..."
+socorro collector2015 \
+    --services.services_controller='[{"name": "collector", "uri": "/submit", "service_implementation_class": "socorro.collector.wsgi_breakpad_collector.BreakpadCollector2015"}, {"name": "pellet_stove", "uri": "/pellet/submit", "service_implementation_class": "socorro.collector.wsgi_generic_collector.GenericCollector"}]' \
+    --services.collector.storage.crashstorage_class=socorro.external.crashstorage_base.PolyCrashStorage \
+    --services.collector.storage.storage_classes='socorro.external.fs.crashstorage.FSPermanentStorage, socorro.external.rabbitmq.crashstorage.RabbitMQCrashStorage' \
+    --services.collector.storage.storage0.crashstorage_class=socorro.external.fs.crashstorage.FSPermanentStorage \
+    --services.collector.storage.storage1.crashstorage_class=socorro.external.rabbitmq.crashstorage.RabbitMQCrashStorage \
+    --services.pellet_stove.storage.crashstorage_class=socorro.external.fs.crashstorage.FSPermanentStorage \
+    --services.pellet_stove.storage.fs_root=./submissions \
+    --resource.rabbitmq.host=localhost \
+    --secrets.rabbitmq.rabbitmq_user=guest \
+    --secrets.rabbitmq.rabbitmq_password=guest \
+    --resource.rabbitmq.virtual_host=/ \
+    --resource.rabbitmq.transaction_executor_class=socorro.database.transaction_executor.TransactionExecutor \
+    --web_server.wsgi_server_class=socorro.webapi.servers.CherryPy \
+    --web_server.ip_address=0.0.0.0 \
+    > collector.log 2>&1 &
+socorro processor \
+    --admin.conf=./config/processor.ini \
+    --resource.rabbitmq.host=$rmq_host \
+    --secrets.rabbitmq.rabbitmq_user=$rmq_user \
+    --secrets.rabbitmq.rabbitmq_password=$rmq_password \
+    --resource.rabbitmq.virtual_host=$rmq_virtual_host \
+    --resource.postgresql.database_hostname=$database_hostname \
```

Comment:
If I'm correct this variable uses the test database by default when running tests. So I think the ``setupdb`` you do earlier do not impact the same database.